src/skills-builder/skills-builder-modal/select-preferences/JobTitleSelect.jsx

```diff
@@ -52,6 +52,7 @@ const JobTitleSelect = () => {
           {formatMessage(messages.jobTitlePrompt)}
         </h4>
         <InstantSearch searchClient={searchClient} indexName={getConfig().ALGOLIA_JOBS_INDEX_NAME}>
+          <Configure filters="b2c_opt_in:true" />
```

Comment:
Is this the first selection box? Like, where we ask what job the person _currently_ has? If so, I don't think we should be filtering here. We should only filter on the when we ask "what are you interested in pursuing". Otherwise, we might not be able to capture the user's information, because their current job might not be in our list.